src/ProductConstructionService/ProductConstructionService.BarViz/ProductConstructionService.BarViz.csproj

```diff
@@ -15,6 +15,16 @@
     <PackageReference Include="Microsoft.FluentUI.AspNetCore.Components" />
     <PackageReference Include="Microsoft.FluentUI.AspNetCore.Components.Icons" />
     <PackageReference Include="TextCopy" />
+    
+    <!-- Pins that can go away once we upgrade to .NET 9.0 centrally -->
```

Comment:
do we have plans to move to 9.0 since it's not LTS?

Reply:
I tried to move the repo to 9.0 and it's problematic with darc still targeting 6.0 (which I don't know if we can bump).

However this PR shows that Blazor WASM compiled by 9.0 targeting 8.0 does not work well if it has libraries inside targeting older. I found quite a few threads on GitHub about problems like these.